pkg/webhook/router.go

```diff
@@ -34,11 +35,11 @@ func Register(mgr controllerruntime.Manager, webhookList ...Webhook) error {
 		return nil
 	}
 
-	s := mgr.GetWebhookServer()
-	for _, wh := range webhookList {
-		s.Register(wh.GetPath(), &webhook.Admission{
+	whServer := mgr.GetWebhookServer()
```

Comment:
please, don't rename variables: it's not a matter of the PR.

Reply:
Not in the scope you are right.

Just a general remark, don't you think that names like "wh" slow down understanding of the code? Maybe I am obsessed with uncle bob's clean code! :-D

Reply:
Uncle Bob is always right, mine is a heritage from other Go codebases where they used to assign really short variables, as you can notice in `mgr` that is easy to understand as `manager`.

I thought the same for `wh` (`webhook`) and `s` as `server` but seems I failed (__but at what cost?__).

Definitely, we have also to take care of the developer experience in order to onboard as fast as possible other people, so I'd raise an Issue to discuss this in-depth since inclusivity can start also from variables naming!